tests/UI/tools/stepIdentifiersChecker.ts

```diff
@@ -0,0 +1,190 @@
+import fs from 'fs';
+
+type JsonReportTest = {
+  context?: string,
+  file: string,
+  pending: boolean,
+  fullTitle: string
+}
+type JsonReportSuite = {
+  file: string,
+  tests: Array<JsonReportTest>
+  suites: Array<JsonReportSuite>
+}
+type JsonReportResult = {
+  suites?: Array<JsonReportSuite>
+}
+type JsonReport = {
+  results: Array<JsonReportResult>
+}
+
+// Get mochawesome report
+const reportPath: string = process.env.REPORT_PATH || './mochawesome-report/mochawesome.json';
+const jsonFile: string = fs.readFileSync(reportPath).toString();
+
+/**
+ * Get all tests from json file
+ * @param jsonFile
+ * @returns {[]}
+ */
+const getAllTests = (jsonFile: string): JsonReportTest[] => {
+  // Parse json report
+  const jsonReport: JsonReport = JSON.parse(jsonFile);
+  const parentSuites = jsonReport.results[0].suites;
+
+  let allTests: JsonReportTest[] = [];
+
+  // eslint-disable-next-line no-restricted-syntax
+  if (parentSuites !== undefined) {
```

Comment:
```suggestion
  if (parentSuites) {
```

should probably be enough?

Reply:
:+1: